src/components/ComponentPreview.js

```diff
@@ -3,55 +3,49 @@ import PropTypes from 'prop-types';
 import IframeResizer from 'iframe-resizer-react';
 import DevDocLinks from './DevDocLinks';
 
-class ComponentPreview extends React.Component {
-  constructor(props) {
-    super(props);
-  }
+const ComponentPreview = (props) => {
+  const {
+    componentName,
+    hasReact,
+    hasAngular,
+    hasHTML,
+    componentType,
+    minHeight,
+    maxHeight,
+    maxWidth,
+    allowScrolling,
+  } = props;
```

Comment:
I would consider destructing the props directly would be a better approach just as a better practice.

```suggestion
const ComponentPreview = ({
    componentName,
    hasReact,
    hasAngular,
    hasHTML,
    componentType,
    minHeight,
    maxHeight,
    maxWidth,
    allowScrolling,
  }) => {
```

Reply:
Love this! Less code.